Solutions/VCenter/Data Connectors/Connector_Syslog_vcenter.json

```diff
@@ -0,0 +1,138 @@
+        }
+        ], 
+            "dataTypes": [ 
+            { 
+               "name": "vCenter_CL", 
+               "lastDataReceivedQuery": "vCenter \n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)" 
+            } 
+        ], 
+             "connectivityCriterias": [ 
+                { 
+                    "type": "IsConnectedQuery", 
+                    "value": [ 
+                    "vCenter \n      | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(7d)" 
+             ] 
+                } 
+            ], 
+            "availability": { 
+            "status": 1,
+            "isPreview": true 
+            }, 
+            "permissions": { 
+            "resourceProvider": [ 
+                   { 
+                       "provider": "Microsoft.OperationalInsights/workspaces", 
+                       "permissionsDisplayText": "write permission is required.", 
+                       "providerDisplayName": "Workspace", 
+                       "scope": "Workspace", 
+                       "requiredPermissions": { 
+                        "write": true,
+                        "read": true,
+                        "delete": true
+                    }                    
+                },
+                {
+                    "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                    "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                    "providerDisplayName": "Keys",
+                    "scope": "Workspace",
+                    "requiredPermissions": {
+                        "action": true
+                    }
+                } 
+            ],
+            "customs": [
+                {
+                    "name": "vCenter",
+                    "description": "must be configured to export logs via Syslog"
+                }
+            ]
+    }, 
+            "instructionSteps": [
+                { 
+                    "title": "", 
+                    "description": "This data connector depends on a parser (based on a Kusto Function) to work as expected. You have 2 options to get this parser into workspace\n\n> 1. If you have installed this connector via vCenter solution in ContentHub then navigate to parser definition from your workspace (Logs --> Functions --> vCenter --> Load the function code) to add your vCenter device list in the query and save the function.\n\n> 2. If you have not installed the vCenter solution from ContentHub then [Follow the steps]() to use the Kusto function alias, **vCenter**",
```

Comment:
This instruction requires modification as Meraki required the customer to add the Device Name but vCenter connector doesn't.
Also, Follow the steps text is missing the link.

Reply:
Instructions are not correct. In case we install the solution from Content Hub, there is not need to save the Parser again. This was required in Meraki because we were adding device names which is not the case with vCenter.
![image](https://user-images.githubusercontent.com/10944026/175877933-6c83ccf4-28cf-47a4-ab42-944e6ca3cf0f.png)

Link is not updated yet.

Reply:
link updated